x-pack/plugins/security/public/analytics/analytics_service.ts

```diff
@@ -58,15 +58,16 @@ export class AnalyticsService {
     this.securityFeaturesSubscription = this.securityLicense.features$
       .pipe(
         filter(({ allowLogin }) => allowLogin),
-        throttleTime(5000)
+        throttleTime(5000),
+        mergeMap(async () => {
```

Comment:
This could be replaced by a `switchMap` as the async callback only depends on the result of the latest invocation.

Reply:
Addressed in [`0fab35e` (#169522)](https://github.com/elastic/kibana/pull/169522/commits/0fab35e43a6392ec41e358d0cd95140323fac58a)